internal/core/metadata/utils.go

```diff
@@ -15,6 +15,7 @@ package metadata
 
 import (
 	"github.com/edgexfoundry/edgex-go/internal/core/data/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
```

Comment:
Group your imports; separate with system imports first.

Reply:
- Done ✅ 
- Please let me know if we're good with following the following convention:
`standard library imports` -> `blank line` -> `third party imports` -> `blank line` -> `edgex imports`